src/SDKs/Search/Management/Management.Search/Generated/ServicesOperations.cs

```diff
@@ -935,6 +935,202 @@ public async Task<AzureOperationResponse<IEnumerable<SearchService>>> ListByReso
             return _result;
         }
 
+        /// <summary>
+        /// Gets a list of all Search services in the given subscription.
+        /// <see href="https://aka.ms/search-manage" />
+        /// </summary>
+        /// <param name='searchManagementRequestOptions'>
+        /// Additional parameters for the operation
+        /// </param>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async Task<AzureOperationResponse<IEnumerable<SearchService>>> ListBySubscriptionWithHttpMessagesAsync(SearchManagementRequestOptions searchManagementRequestOptions = default(SearchManagementRequestOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Looks like we're a bit behind on updating the management SDK from the spec. This was added to the spec about a month ago. It's going to need test coverage (we can sync offline about who should do that work).